File: Java/CheckPangram.java
package Java;
//Panagram all letters from a to z is present in the given string
public class CheckPangram {
    public static void main(String[] args) {
        String input = "Quick Brown fox jumps over the lazy dog";
        System.out.println("The given String is Panagram: "+isPangram(input));
    }
    static boolean isPangram(String s){
        s = s.toLowerCase();
        if(s.length() < 26){
            return false;
        }
        for(char i = 'a';i <= 'z';i++){
            if(s.indexOf(i) < 0){//Which mean the letter is not present in string, it will return -1
                return false;
            }
        }
        return true;
    }
}

File: Python/Check_pangram.py
s = "a quick brown fox jumps over the lazy dog"
print(ord('a'))
print(ord('z'))
def isPangram(s):
    if len(s) < 26:
        return False
    s = s.lower()
    for i in range(ord('a'),ord('z')+1):
        if chr(i) not in s:
            return False
    return True
print(isPangram(s))

def usingSet(s):
    return len(set(s)) >= 26
print(usingSet(s))